src/settings/AcquisitionUnitsList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import {
+  Pane,
+} from '@folio/stripes/components';
+
+const AcquisitionUnitsList = () => {
+  return (
+    <Pane
+      id="pane-ac-units-list"
+      defaultWidth="fill"
+      paneTitle="Acquisition units"
```

Comment:
Should it be translated?

Reply:
yes, but in next story, I'm pretty sure structure and root components will be changed